public/index.html

```diff
@@ -369,6 +369,7 @@
 // Get first object with views > 100 or throw error
 data.posts.where('views', '>', 100).firstOrFail())</code></pre>
       <h4>Filtering</h4>
+      <p>Add `filter_index` to the desired field to use methods below</p>
```

Comment:
Use this instead of paragraph(also change this for order_index)

`<pre data-type="socket"><code class="lang-yaml">classes:
posts:
  title:
    filter_index: true</code></pre>`